docs/tutorials/typescript/01-intro-typescript.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_label: Writing policies in TypeScript/JavaScript
+sidebar_position: 010
+title: Writing policies in TypeScript/JavaScript
+description: A tutorial introduction to writing policies in TypeScript/JavaScript.
+keywords: [kubewarden, kubernetes, writing policies in TypeScript, writing policies in JavaScript]
+doc-type: [tutorial]
+doc-topic: [kubewarden, writing-policies, typescript, javascript, introduction]
+doc-persona: [kubewarden-policy-developer]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/tutorials/writing-policies/typescript/intro-typescript"/>
+</head>
+
+:::note
+TypeScript/JavaScript support for WebAssembly is rapidly evolving.
+This page was last revised in September 2025.
+:::
+
+Kubewarden uses [Javy](https://github.com/bytecodealliance/javy) (a Bytecode Alliance project) to build WebAssembly binaries from JavaScript.
+
+> Javy takes your JavaScript code and executes it in a WebAssembly context. 
+>
+> It features an embedded QuickJS engine, compiled to WebAssembly, that can execute JavaScript. 
+>
+> The project provides both a CLI and a set of APIs for embedding and customizing the behavior when running JavaScript in WebAssembly.
+
+The Kubewarden project currently uses Javy for these reasons:
+
+- Mature JavaScript engine (QuickJS) compiled to WebAssembly
+- Support for [waPC](https://wapc.io) through custom host functions
```

Comment:
This is not correct, we're not using waPC as part of the JS SDK. The policy we're writing is targeting our WASI interface